### ISSUE TITLE
Support for reactions & edited comments

### ISSUE THREAD

**Issue by Markersx** [Scarsz/DiscordSRV#89]
Wood bee supper coal
Would be super cool (edited)

**Comment by Tisawesomeness** [comment]
Found it! It's :memo:

**Comment by Markersx** [comment]
Those are not the duplicates I was thinking about that would be too easy I don't know why there is 2 of them.

**Comment by Scarsz** [comment]
☎️ ☎️ also duplicated

**Comment by Markersx** [comment]
They are not right next to each other

**Comment by Vankka2** [comment]
There is 3 💩

**Comment by Markersx** [comment]
The duplicate I was thinking of is :couple: can you find it?

**Comment by xXBuilderBXx** [comment]
There is 2 duplicates i have found 🚗 🚗 and the 2 sails above the cars :/

**Comment by Scarsz** [comment]
Right, what was actually expected out of this issue? I don't really know what you mean by `supporting edited messages`; emojis are already supported.

**Comment by xXBuilderBXx** [comment]
I think he meant sending messages to minecraft that have been edited from discord but i dont see a viable use for this
